showcase/src/app/demo/routes.ts

```diff
@@ -0,0 +1,143 @@
+import { Ng2StateDeclaration } from "@uirouter/angular";
+import { DemoActionBarComponent } from "./action-bar";
+import { DemoBreadcrumbComponent } from "./breadcrumb";
+import { DemoButtonComponent } from "./button";
+import { DemoCardComponent } from "./card";
+import { DemoCollapsibleComponent } from "./collapsible";
+import { DemoColorsComponent } from "./colors";
+import { DemoDatePickerComponent } from "./date-picker";
+import { DemoDateRangePickerComponent } from "./date-range-picker";
+import { DemoDropdownComponent } from "./dropdown";
+import { DemoFooterComponent } from "./footer";
+import { DemoHeaderComponent } from "./header";
+import { DemoExampleViewerComponent } from "./example-viewer";
+import { DemoKeyboardDirectivesComponent } from "./keyboard-directives";
+import { DemoLanguageSelectorComponent } from "./language-selector";
+import { DemoLogoutComponent } from "./logout";
+import { DemoMenuComponent } from "./menu";
+import { DemoPaginationComponent } from "./pagination";
+import { DemoPrettyPrintComponent } from "./pretty-print";
+import { DemoSliderComponent } from "./slider";
+import { DemoSidebarComponent } from "./sidebar";
+import { DemoTableComponent } from "./table";
+import { DemoToastComponent } from "./toast";
+import { DemoTypographyComponent } from "./typography";
+
+export const DEMO_STATES: Ng2StateDeclaration[] = [
+	{ name: "demo", url: "^/demo", abstract: true, parent: "app" },
+	{
+		name: "demo.action-bar",
+		url: "/action-bar",
+		views: { "@": { component: DemoActionBarComponent } }
```

Comment:
Can't we simply declare our states as before like this:
```typescript
{
  name: "demo.action-bar",
  url: "/action-bar",
  component: DemoActionBarComponent
},
```
? 😊

Reply:
Nop, we need the `views: { "@": { component: HomeComponent } }` construct, see my previous above.